android/app/src/main/java/org/chrome_on_device_ml/experiments/BertExperiment.java

```diff
@@ -0,0 +1,109 @@
+/* Copyright 2020. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+package org.chrome_on_device_ml.experiments;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.HandlerThread;
+import android.os.Message;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.chrome_on_device_ml.ml.LoadDatasetClient;
+import org.chrome_on_device_ml.ml.QaAnswer;
+import org.chrome_on_device_ml.ml.QaClient;
+
+public class BertExperiment{
+  private static final String TAG = "CDML_MobileBert";
+  private final Context context;
+  private Handler handler;
+  private Handler UIHandler;
+
+  private LoadDatasetClient datasetClient;
+  private QaClient qaClient;
+  private ArrayList<Double> timing;
+
+  public BertExperiment(Context context, Handler handler) {
+    this.context = context;
+    this.UIHandler = handler;
+
+    this.datasetClient = new LoadDatasetClient(this.context);
+
+    HandlerThread handlerThread = new HandlerThread("BertExp");
+    handlerThread.start();
+
+    this.handler = new Handler(handlerThread.getLooper());
+    this.qaClient = new QaClient(this.context);
+    this.timing = new ArrayList<Double>();
+  }
+
+  public void initialize() {
+    handler.post(
+      () -> {
+        qaClient.loadModel();
+        qaClient.loadDictionary();
+      });
+  }
+
+  public void close() {
+    handler.post(
+      () -> {
+        qaClient.close();
+      });
+  }
+
+  /** Evaluates Bert model with contents and questions. */
```

Comment:
single-line comments should just be //

Reply:
I followed examples like [this code]?(https://github.com/tensorflow/examples/blob/6b5ec484222f0e81837f11591f7eea25d52ed158/lite/examples/text_classification/android/app/src/main/java/org/tensorflow/lite/examples/textclassification/TextClassificationClient.java#L68)
Please let me know if I should change them to //

Reply:
I resolve this since the code the I reused use the same.

Reply:
most java style within google uses // for documentation comments (i.e. not javadoc)

Reply:
Okay, I'll change them.

Reply:
Changed all comments to // except the class description.